kernel/src/engine/mod.rs

```diff
@@ -3,7 +3,7 @@
 //! related modules for more information.
 
 #[cfg(feature = "arrow-conversion")]
-pub(crate) mod arrow_conversion;
+pub mod arrow_conversion;
```

Comment:
Oh this was probably why my usage in the past was unnecessarily hard, these weren't exposed